app/templates/admin/userManagement.html

```diff
@@ -166,7 +167,39 @@ <h5>Programs</h5>
       </div>
     </div>
   </div>
-  
+{% endif %}
+{% if g.current_user.isProgramManager() and not g.current_user.isCeltsAdmin %}
```

Comment:
Two things here. 

1. You shouldn't be able to edit all programs just because you're a program manager for one program.
2. We shouldn't duplicate all of this html between the admin vs the program manager cases. Look at using a macro to emit the markup that is shared.